Apps/Playground/Android/app/CMakeLists.txt

```diff
@@ -33,8 +33,14 @@ set(BABYLON_NATIVE_PLAYGROUND_DIR "${CMAKE_CURRENT_LIST_DIR}/../..")
 add_subdirectory(${BABYLON_NATIVE_PLAYGROUND_DIR}/../../ ${BABYLON_NATIVE_PLAYGROUND_DIR}/../../build/Android_${CMAKE_ANDROID_ARCH_ABI}/)
 
 add_library(BabylonNativeJNI SHARED
+    ${BABYLON_NATIVE_PLAYGROUND_DIR}/Shared/Playground/GraphicsApiInfo.cpp
     src/main/cpp/BabylonNativeJNI.cpp)
 
+if(NOT GRAPHICS_API)
+    message(FATAL_ERROR "GRAPHICS_API must be set")
+endif()
```

Comment:
The top-level CMakeLists.txt for this repo is added as a subdirectory on Android, so the `GRAPHICS_API` CMake variable is not set in the scope of *this* CMakeLists.txt. This check makes sure the `GRAPHICS_API` variable has been set in the CMake command used to configure the build, e.g. `cmake -B build -D GRAPHICS_API=OpenGL`. The Android does this in it's gradle build config.

Maybe we don't need it?